src/tests/ftest/util/command_utils.py

```diff
@@ -518,12 +531,14 @@ def __init__(self, job, path="", subprocess=False):
             "/run/orterun", "orterun", job, path, subprocess)
 
         self.hostfile = FormattedParameter("--hostfile {}", None)
-        self.processes = FormattedParameter("-np {}", 1)
-        self.display_map = FormattedParameter("--display-map", True)
+        self.processes = FormattedParameter("--np {}", 1)
+        self.display_map = FormattedParameter("--display-map", False)
         self.map_by = FormattedParameter("--map-by {}", "node")
         self.export = FormattedParameter("-x {}", None)
         self.enable_recovery = FormattedParameter("--enable-recovery", True)
         self.report_uri = FormattedParameter("--report-uri {}", None)
+        self.allow_run_as_root = FormattedParameter("--allow-run-as-root", None)
```

Comment:
This should have a default of `False`:
```suggestion
        self.allow_run_as_root = FormattedParameter("--allow-run-as-root", False)
```